lib/main.dart

```diff
@@ -1,36 +1,51 @@
 import 'package:firebase_core/firebase_core.dart';
 import 'package:flutter/material.dart';
-import 'package:iiit_sonepat_stable/constants/router.dart';
+
 import 'package:iiit_sonepat_stable/constants/routes.dart';
 import 'package:iiit_sonepat_stable/ui/Auth/SignUp.dart';
+import 'package:iiit_sonepat_stable/ui/Login_Page/Landing.dart';
+import 'package:iiit_sonepat_stable/ui/Login_Page/SplashScreen.dart';
 
 void main() async {
-  WidgetsFlutterBinding.ensureInitialized();
-  await Firebase.initializeApp();
-  runApp(MyApp());
+  Map<int, Widget> op = {1: Landing(), 2: Landing()};
+  // WidgetsFlutterBinding.ensureInitialized();
+  // await Firebase.initializeApp();
+  runApp(
+    MaterialApp(
+      home: CustomSplash(
+        imagePath: 'assets/images/IIIT Sonepat Logo.png',
+        backGroundColor: Colors.white,
+        animationEffect: 'zoom-in',
+        logoSize: 200,
+        home: Landing(),
+        duration: 4000,
+        type: CustomSplashType.StaticDuration,
+        outputAndHome: op,
+      ),
+    ),
+  );
 }
 
-class MyApp extends StatelessWidget {
```

Comment:
Hey @SanskarModi22 , i think this should not be removed also, call the MaterialApp under this widget only.